examples/hello_services/HelloServices/main.m

```diff
@@ -0,0 +1,14 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <UIKit/UIKit.h>
+#import <Flutter/Flutter.h>
+
+int main(int argc, char * argv[]) {
+    FlutterInit(argc, (const char**)argv);
```

Comment:
Yeah, I didn't do that because `[NSProcessInfo arguments]` gives you the arguments in an NSArray and then it's a bunch for work to convert that back into argc and argv.  I looked at plumbing the NSArray all the way down to base::CommandLine, but that was ugly too...  It might be worth it to get ride of this line, however.